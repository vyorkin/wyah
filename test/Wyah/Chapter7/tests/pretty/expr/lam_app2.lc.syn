EApp (EApp (EApp (EVar (Var "n")) (ELam (Var "g") (EApp (EVar (Var "h")) (EApp (EVar (Var "g")) (EVar (Var "f")))))) (ELam (Var "u") (EVar (Var "x")))) (EVar (Var "I"))
