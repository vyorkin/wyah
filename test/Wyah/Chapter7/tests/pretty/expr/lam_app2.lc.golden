((n (\g -> h (g f))) (\u -> x)) I